### PR TITLE
maliput_cargo: Traffic lights example.

### PR DESCRIPTION
**Pull request by francocipollone**
# 🎉 New feature

Closes #<NUMBER>

## Summary
- Adds traffic light example 

## Test it
cargo run --bin traffic_lights

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [x] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if it affects the public API)

### CODE REVIEW

File: maliput_cargo/src/traffic_lights.rs [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    println!("linear_tolerance: {}", road_geometry.linear_tolerance());
+    println!("angular_tolerance: {}", road_geometry.angular_tolerance());
+    println!("num_junctions: {}", road_geometry.num_junctions());
+
+    let lanes = road_geometry.get_lanes();
+    println!("num_lanes: {}", lanes.len());
+    println!("lanes: ");
+    for lane in lanes {
+        println!("\tlane id: {}", lane.id());
+    }
```
Comment (agalbachicar) [inline maliput_cargo/src/traffic_lights.rs]: Shall we skip this in favor of a reduced and focused example?

**Review by agalbachicar — CHANGES_REQUESTED** [review]
Just one comment to go.

File: maliput_cargo/src/traffic_lights.rs (hunk 1 above)
Comment (francocipollone) [reply maliput_cargo/src/traffic_lights.rs]: Done.

**Review by agalbachicar — APPROVED** [review]
LGTM

**Pull request merged**